crates/red_knot_python_semantic/resources/mdtest/subscript/string.md

```diff
@@ -20,6 +20,38 @@ b = s[-8]  # error: [index-out-of-bounds] "Index -8 is out of bounds for string
 reveal_type(b)  # revealed: Unknown
 ```
 
+## Slices
```

Comment:
Similar here. These tests are not exhaustive in terms of checking slicing functionality, as we do that in the unit tests already. Let me know if you think otherwise.

Reply:
No, that seems fine!